canonical-websites/jaas.ai
# Fix tags

## Done

- Correctly get and display the charm tags.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Do a search, the results should have tags.
